CefSharp/BindingOptions.cs

```diff
@@ -29,5 +29,12 @@ public static BindingOptions DefaultBinder
         /// for logging calls (from js) to .net methods.
         /// </summary>
         public IMethodInterceptor MethodInterceptor { get; set; }
+
+        /// <summary>
+        /// Interceptor used for intercepting calls to the target object property. For instance, can be used 
```

Comment:
> Interceptor used for intercepting calls to the target object property

Interceptor used for intercepting get/set calls to the target object property

Reply:
Incorporated the comment.